### DISCUSSION TITLE
What is the difference between Minestom's TimeUnit client tick and server tick

### DISCUSSION THREAD

**Discussion by emortaldev** [Minestom/Minestom#1334] [Help] [answered]
TimeUnit.CLIENT_TICK and TimeUnit.SERVER_TICK

**Comment by GoldenStack** [comment, marked as answer]
The server doesn't know what the client is doing, so it has to assume that it uses 50ms ticks.
Minestom can be configured to have a different TPS, so the tick time is calculated from the internal value.

Refer to the source;
https://github.com/Minestom/Minestom/blob/fe1bfaee936e3cc8ecf496087e5d19232727f5e6/src/main/java/net/minestom/server/utils/time/Tick.java#L13-L22
(the definitions in `TimeUnit` just refer to these)
